integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/GraphPagingIT.java

```diff
@@ -461,38 +464,6 @@ public void should_trigger_global_timeout_async() throws InterruptedException {
     }
   }
 
-  @Test
-  public void should_trigger_global_timeout_async_after_first_page() throws InterruptedException {
```

Comment:
Why do you want to remove this test? It exercises the global timeout for async paging.

Reply:
Discussed in Slack: TLDR is this test is too broken to be kept.